src/parser.rs

```diff
@@ -5337,6 +5337,24 @@ impl<'a> Parser<'a> {
             vec![]
         };
 
+        let range_by = {
+            let mut align: Option<Value> = None;
+            let mut fill: Option<Ident> = None;
+            for _ in 0..2 {
+                if self.parse_keyword(Keyword::ALIGN) {
+                    align = Some(self.parse_value()?);
+                }
+                if self.parse_keyword(Keyword::FILL) {
+                    fill = Some(self.parse_identifier()?);
+                }
+            }
```

Comment:
Maybe we should report error for something like `FILL ... FILL ...` or `ALIGN ... ALIGN ...`?